packages/crayons-core/src/components/icon/readme.md

```diff
@@ -1,18 +1,25 @@
 # Icon (fw-icon)
-fw-icon displays an icon-sized image that imparts meaning to the component it is associated with.
 
-## Demo
+"fw-icon/FwIcon" is a renderer of SVG file that displays an icon-sized image that imparts meaning to the HTML component it is associated with.
+Following features are available as part of the implementation of the fw-icon.
+
+1. Enable fw-icon as an optimized renderer for SVG with built-in functions such as Intersection-Observer and Fetch-API Memoization.
+   Go through the docs to understand the various props it supports.
+2. React-Support as part of Crayons Release. 
```

Comment:
You can remove this statement, since by default we provide `React` wrapper for all the components.

Reply:
Fixed